src/imports/logger.js

```diff
@@ -1,6 +1,13 @@
 import winston from 'winston';
 
-// TODO: Configure logger
-const logger = winston;
+const logger = new winston.Logger;
+
+logger.add(winston.transports.Console, {
+  colorize: true,
+  level: 'debug',
+  prettyPrint: true,
+  silent: false,
+  timestamp: false,
```

Comment:
I think we should timestamp so that we can see how far apart things are (how long things are taking to run). This will be important to have when servers and other things are writing output to the logger.